src/error.js

```diff
@@ -234,6 +239,28 @@ export function isCancellation(errorOrMessage) {
   return false;
 }
 
+export function blockByConsent() {
+  return new Error(BLOCKBYCONSENT);
+}
+
+/**
+ * @param {*} errorOrMessage
```

Comment:
I tried `{!Error|string}`, but switched back to `{*}`. 
Looks like I can't use `{!Error|string}` because `throw` can throw anything.  get this error
```
src/custom-element.js:531: ERROR - actual parameter 1 of module$src$error.isBlockedByConsent does not match formal parameter
found   : *
required: (Error|string)
        if (!isBlockedByConsent(reason)) {
```